satellite/console/consoleweb/server.go

```diff
@@ -131,7 +173,9 @@ func (server *Server) appHandler(w http.ResponseWriter, req *http.Request) {
 	header.Set("Content-Type", "text/html; charset=UTF-8")
 	header.Set("Content-Security-Policy", strings.Join(cspValues, "; "))
 
-	http.ServeFile(w, req, filepath.Join(server.config.StaticDir, "dist", "index.html"))
+	if server.templates.index == nil || server.templates.index.Execute(w, nil) != nil {
```

Comment:
how template can be nil?

Reply:
Yeah, this should never be nil.

Reply:
server.templates.index will be nil when index.html file is missing. This file is generated by `npm run build` command, which could be not called in some cases. For example in our integration tests

Reply:
Can we use some non-generated index.html that will build the javascript things?

Reply:
whole dist folder is generated by npm. And we don't have such folder in github sources. To fix it we can add `web/satellite/dist` folder and `web/satellite/dist/index.html` file to git. But i'm not sure that it is a good idea

Reply:
Let's add a `index.html` that basically contains a text with something like "run npm with XYZ to generate the necessary files"

Reply:
@egonelbre may it be done in separate pr also?

Reply:
Yeah, sure.